featuretools/tests/primitive_tests/test_transform_primitive.py

```diff
@@ -174,6 +175,40 @@ def test_is_quarter_start():
     np.testing.assert_array_equal(iqs_bools, correct_bools)
 
 
+def test_part_of_day():
+    pod = PartOfDay()
+    dates = pd.Series(
+        [
+            datetime(2020, 1, 11, 1, 2, 1),
+            datetime(2021, 3, 31, 4, 2, 1),
+            datetime(2020, 3, 4, 6, 2, 1),
+            datetime(2020, 3, 4, 8, 2, 1),
+            datetime(2020, 3, 4, 11, 2, 1),
+            datetime(2020, 3, 4, 14, 2, 3),
+            datetime(2020, 3, 4, 17, 2, 3),
+            datetime(2020, 2, 2, 20, 2, 2),
+            np.nan,
+        ]
+    )
+    answer = pod(dates)
+    correct_answer = pd.Series(
+        [
+            "midnight",
+            "dawn",
+            "early morning",
+            "late morning",
+            "noon",
+            "afternoon",
+            "evening",
+            "night",
+            np.nan,
+        ]
+    )
+    answer = [i if not pd.isna(i) else None for i in answer]
+    correct_answer = [i if not pd.isna(i) else None for i in correct_answer]
+    np.testing.assert_array_equal(answer, correct_answer)
```

Comment:
a nit, but convention is to compare `expected` and `actual`, not `answer` and `correc_answer`